src/routes/dockerflow.js

```diff
@@ -8,7 +8,6 @@
 
```

Comment:
This change is https://github.com/firefox-devtools/profiler-server/pull/16, please do not look at it.